servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -657,6 +662,34 @@ public ZMSImpl() {
         // system disabled from UserAuthority
 
         initializePrincipalStateUpdater();
+        
+        // load the DomainChangePublisher
+        
+        loadDomainChangePublisher();
+    }
+
+    void loadDomainChangePublisher() {
+        String topicNames = System.getProperty(ZMS_PROP_DOMAIN_CHANGE_TOPIC_NAMES, "");
+        for (String topic : topicNames.split(",")) {
+            if (!topic.isEmpty()) {
```

Comment:
instead of just calling topic.trim() on line 678, it's better that we trim it first, then check if it's empty, otherwise, we'll support values like: "message1, , message2" and we'll end up calling createPublisher with an empty string.